### ISSUE TITLE
Incorrect installation instructions

### ISSUE THREAD

**Issue by igor-savin-ht** [bogus34/bookshelf-schema#15]
"bookshelf.plugin require('bookshelf-schema')()" doesn't seem to be valid Javascript.

**Comment by bogus34** [comment]
Yep, because it's CoffeeScript. But you're right, probably I should state it explicitly and/or add JS version.

**Comment by igor-savin-ht** [comment]
Wow, someone still uses CoffeeScript!

**Comment by bogus34** [comment]
What's wrong with it?
